packages/react-tooltip/src/components/TooltipProvider/useTooltipManager.ts

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { ShowTooltipArgs, TooltipManager, TooltipTriggerReason } from '@fluentui/react-tooltip-trigger';
+
+const useTimeout = () => {
+  type TimeoutState = {
+    id: number | undefined;
+    set: (fn: () => void, delay: number) => void;
+    clear: () => void;
+  };
+
+  const state = React.useRef<TimeoutState>({
```

Comment:
will this state ever change ? if it doesn't it might make more sense to just use `React.useState()` and never call the dispatch

Reply:
You will also never re-create the initial value ever again